cpp/include/cuspatial/spatial_join.hpp

```diff
@@ -22,6 +22,11 @@
 
 namespace cuspatial {
 
+/**
+ * @addtogroup spatial_join
+ * @{
+ */
+
```

Comment:
I can't comment on it since it's not part of the diff, but I don't think the return statement on line 55 is going to render the way that you want. It'll just flatten it all into a single line. That may be something that you improve in a future PR, though, instead of trying to fix it here.